README.md

```diff
@@ -1 +1,97 @@
-# Hello-Microverse-Repository
\ No newline at end of file
+# HTML & CSS3 Course
+
+If you are not familiar with linters, read [root level README](../README.md).
+
```

Comment:
- [ ] Remember that when we create a Pull Request, the person that it's going to review your proyect needs to know on the first view the changes that you're doing on that PR. Kindly add a description of the changes that you are making on this PR, you can use the requirements of this activity for help 💯